### ISSUE TITLE
Error in shorthand-property-no-redundant-values

### ISSUE THREAD

**Issue by dzhiriki** [stylelint/stylelint#1213]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Yet another bug which I found in 7ba84e9.

> Which rule, if any, is this issue related to?

`shorthand-property-no-redundant-values`

> What CSS is needed to reproduce this issue?

```css
.item {
	border-color: #FDB00A transparent transparent;
}
```

Color should be in upper case

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "shorthand-property-no-redundant-values": true
  }
}
```

> Which version of stylelint are you using?

e.g. `6.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
test.css
4:2  ✖  Unexpected longhand value '#FDB00A transparent          shorthand-property-no-redundant-values
         transparent' instead of '#FDB00A transparent
         transparent'
```

**Comment by evilebottnawi** [comment]
Ref: https://github.com/stylelint/stylelint/pull/1214
